Kitodo-API/src/main/java/org/kitodo/api/dataeditor/rulesetmanagement/ComplexMetadataViewInterface.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataeditor.rulesetmanagement;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Provides an interface for the complex-type view service. The complex-type
+ * view service provides a filtered view of a complex meta-data type.
+ */
+public interface ComplexMetadataViewInterface extends MetadataViewInterface {
+    /**
+     * Returns the meta-data keys that the user can still add to the form. The
+     * list contains only meta-data keys that are still allowed at this point,
+     * minus those that are already displayed anyway but have no value yet.
+     *
+     * @param <T>
+     *            the type of meta-data objects
+     * @param entered
+     *            meta-data objects that have already been entered, along with
+     *            their key
+     * @param additionallySelected
+     *            meta-data keys that the user has already selected
+     * @return the meta-data keys that the user can add
+     */
+    <T> Collection<MetadataViewInterface> getAddableMetadatas(Map<T, String> entered,
```

Comment:
There is no plural "metadatas" , its just "metadata"

Reply:
I changed the two.